roles/ovirt-external-providers/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Add external providers
+  ovirt_external_providers:
+    auth: "{{ ovirt_auth }}"
+    name: "{{ item.name }}"
+    state: "{{ item.state }}"
+    type: "{{ item.type }}"
+    url: "{{ item.url }}"
+    username: "{{ item.username }}"
+    password: "{{ item.password }}"
+    tenant: "{{ item.tenant }}"
+    auth_url: "{{ item.auth_url }}"
```

Comment:
I think most of those paramaters are not mandatory, so we should use `default(omit)`